src/Apps/Auction/Components/PricingTransparency.tsx

```diff
@@ -0,0 +1,67 @@
+import { Flex, Sans, Serif } from "@artsy/palette"
+import React from "react"
+
+export const PricingTransparency: React.FC<{}> = () => {
+  return (
+    <Flex pt={3} flexDirection="column">
+      <Serif pb={1} size="4t" weight="semibold" color="black100">
+        Summary
+      </Serif>
+
+      <Flex
+        flexDirection="row"
+        justifyContent="space-between"
+        alignItems="baseline"
+        pb={1}
+      >
+        <div>
```

Comment:
We generally avoid `<div>` these days and use `<Box>` (which is universal / copy / pasteable to RN).

Reply:
Hmm yea, i just copied this directly from our order app so assumed that was intentional to convey that this was strictly a row